app/components/LargeHeader.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { images } from '../assets';
+import { browserHistory } from 'react-router'
 
 
 function LargeHeader() {
+  var submitSearch = function() {
+    browserHistory.push({
+        pathname: '/resources',
+        query: { query: document.getElementById("srch-term").value }
```

Comment:
you can use refs to get the value of inputs and things:
https://facebook.github.io/react/docs/more-about-refs.html